TEAM_STRUCTURES.md

```diff
@@ -194,6 +195,20 @@ The IPFS Cluster Working Group is the team implementing IPFS Cluster.
 - Design and implement IPFS Cluster.
 - Provide knowledge and APIs that organizations with large data sets can use.
 
+### Decentralized Data Stewardship
+
+- **Coordination**: Soon(tm)
```

Comment:
can we have this repo before merging?

Reply:
It sounds like we should resolve the naming conversation above before we create the repo, but I'm happy to have this ASAP. I don't have permissions to create new repos, so when we align on a name, someone else will need to create the repo for me.

If possible (not sure how fine-grained org controls are in GitHub), since I'm going to be the Captain, it would be great to have top-level control of that particular repo once it's created.

Reply:
Also got you a shine tab on the OKR spreadsheet for next quarter! https://docs.google.com/spreadsheets/d/139lROP7-Ee4M4S7A_IO4iIgSgugYm7dct620LYnalII/edit#gid=1521120849